routes/admin.js

```diff
@@ -255,17 +255,6 @@ checkSchema({
     });
 });
 
```

Comment:
```suggestion

router.get('/listInactiveCompany', (req, res) => {
  req.db.query('select name, description, logo from company where active=0;')
    .then(results => {
      res.send(results);
    })
    .catch(err => {
      debug(err);
      res.status(500).send(err);
    });
});
```

We have to list inactive companies for the admin, this was one of the first task that you did.